FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unversioned
+- Adding plist property"FirebaseDeepLinkPasteBoardRetrievalEnabled" to enable/disable fetching dynamic links from PasteBoard.
```

Comment:
Two issues here. Neither directly related to this PR:
1. M74 was deferred to M75 after the CHANGELOG had been updated for it.
2. The current release is 4.1, so the next release should be 4.2.0 and include these top three changes.

Reply:
Done...Please check whether i did it correct.